bus/bus.go

```diff
@@ -1897,7 +1900,7 @@ func (b *bus) multipartHandlerUploadPartPUT(jc jape.Context) {
 	} else if req.ContractSet == "" {
 		jc.Error(errors.New("contract_set must be non-empty"), http.StatusBadRequest)
 		return
-	} else if req.Etag == "" {
+	} else if req.ETag == "" {
```

Comment:
It would be pretty sweet if we could check for the actual etag now. To make sure we end up with the ones we compute and store in the database instead of the ones that gofakes3 adds.

Reply:
added a test for that